store/src/java/com/zimbra/cs/service/util/JWTUtil.java

```diff
@@ -126,7 +129,7 @@ public static final String getSalt(Element soapCtxt, Map engineCtxt) {
                     javax.servlet.http.Cookie cookies[] =  req.getCookies();
                     if (cookies != null) {
                         for (int i = 0; i < cookies.length; i++) {
-                            if (cookies[i].getName().equals(Constants.ZM_JWT_COOKIE)) {
+                            if (cookies[i].getName().equals(ZimbraCookie.COOKIE_ZM_JWT)) {
```

Comment:
can use ZimbraCookie.COOKIE_ZM_JWT.equals() to avoid NPE

Reply:
done